src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java

```diff
@@ -57,9 +57,7 @@
 
 @Transactional
 public class PaymentWorksWebServiceImpl implements PaymentWorksWebService {
-
-	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger
-			.getLogger(PaymentWorksWebServiceImpl.class);
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PaymentWorksWebServiceImpl.class);
 
 	private static final String NEW_VENDOR_REQUEST_PENDING_VENDORS = "new-vendor-requests/";
 	private static final String NEW_VENDOR_REQUEST_UPDATE_STATUS = "new-vendor-requests/bulk/";
```

Comment:
these are URL rest call paths. Perhaps they would be better be handled in another way.  I will put a story in the backlog to review that, and do it consistently with what we are doing amazon and concur.